docs/cookbook/getFirstTuesdayOfMonth.mjs

```diff
@@ -0,0 +1,26 @@
+import assert from "assert";
+
+/**
+ * Gets the first tuesday of the month and returns its date
+ *
+ * @param {Temporal.YearMonth} queriedMonth YearMonth instance to query
+ * @returns {Temporal.DateTime} DateTime Instance which gives first tuesday
```

Comment:
The code as currently written returns a Temporal.Date, and I think that's correct because we don't care about the time of day on Tuesday.

Reply:
Im not sure this is correct, it seems to be returning a DateTime instance for me